apps/blazor-test-app/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "blazor-test-app",
+  "private": true,
+  "author": "Microsoft Teams",
+  "description": "Blazor test app to ensure teams-js changes don't break Blazor app functionality",
+  "version": "1.0.0",
+  "scripts": {
+    "compile": "dotnet build",
```

Comment:
I would suggest `build` instead of `compile` to be consistent across test apps.

Reply:
see mega-comment